sql/updates/world/2015_06_23_00_world_toc.sql

```diff
@@ -0,0 +1,149 @@
+-- Trial of the Champion
+-- by Appled
+-- Thanks to Aokromes for providing sniffs
+
+--
+-- General
+--
+
+-- Disabling mmap for Trial of the Champion instance
+-- creatures are spawned according to sniffs behind an invisible wall and they don't know how to go through it
+-- Edit: Aokromes has pushed this already but disabled whole instance in the same commit
+DELETE FROM `disables` WHERE `entry`='650' AND `sourceType`='2';
+-- DELETE FROM `disables` WHERE `sourceType`='7' AND `entry`='650';
+-- INSERT INTO `disables` (`sourceType`,`entry`) VALUES ('7','650');
+
+-- Spellscript for vehicles' Trample aura
+DELETE FROM `spell_script_names` WHERE `spell_id`='67868' AND `ScriptName`='spell_toc5_trample_aura';
+INSERT INTO `spell_script_names` VALUES ('67868','spell_toc5_trample_aura');
+
+--
+-- Vehicles
+--
+
+-- Variables
+SET @ARGENT_BATTLEWORG_A := 36559; -- cosmetic vehicle (cannot be used)
+SET @ARGENT_BATTLEWORG_H := 36558;
+SET @ARGENT_WARHORSE_A := 36557;
+SET @ARGENT_WARHORSE_H := 35644; -- cosmetic vehicle (cannot be used)
+
+-- Argent Battleworg should not be attackable by players
+UPDATE creature_template SET `unit_flags`=`unit_flags`|256 WHERE `entry`=@ARGENT_BATTLEWORG_A;
+
+-- Argent Battleworg is missing its Trample aura
+UPDATE `creature_template_addon` SET `auras`='67870' WHERE `entry`=@ARGENT_BATTLEWORG_H;
+
+-- Argent Warhorse is missing its vehicle data
+DELETE FROM `npc_spellclick_spells` WHERE `npc_entry` IN (@ARGENT_BATTLEWORG_H,@ARGENT_WARHORSE_A,@ARGENT_WARHORSE_H) AND `spell_id`='67830';
+UPDATE `creature_template` SET `npcflag`='16777216',`speed_run`='1.57143',`spell1`='68505',`spell2`='62575',`spell3`='68282',`spell4`='62552',`VehicleId`='486' WHERE `entry`=@ARGENT_WARHORSE_A;
+UPDATE `creature_template` SET `npcflag`='16777216',`speed_run`='1.57143',`spell1`='68505',`spell2`='62575',`spell3`='68282',`spell4`='62552',`VehicleId`='486' WHERE `entry`=@ARGENT_WARHORSE_H;
```

Comment:
I am confused, why this line when I see this:
SET @ARGENT_WARHORSE_H := 35644; -- cosmetic vehicle (cannot be used)
npc flag 16777216 is for spellclick, so when it cannot be used why this? Please explain

Btw, these NPCs already  has vehicle ID 486